README.md

```diff
@@ -16,6 +16,16 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
 contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
+## E2E
+This project leverages Terraform and the local user's Azure credentials to run an extensive E2E suite.
+
+### Public Cluster/Public DNS Zone
+The process to run an E2E test for public clusters is as follows: 
+1. Run `make clean-public` or `make clean-all` to clear any preexisting Terraform state for the public cluster dev environment.
```

Comment:
`make clean-call` doesn't exist right now

Reply:
How was this resolved @jaiveerk ?

Reply:
removed it as part of my latest commit (just pushed)! I marked it as resolved when I made the change locally to make it easier to figure out which comments I was done addressing.

Reply:
[diff](https://github.com/Azure/aks-app-routing-operator/pull/33/commits/3e098e21b3668f66fb010afaa04b834e47f4226f#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5)

Reply:
yeah seems i was on an old version of the pr! thanks!